test/wpt/status/eventsource.status.json

```diff
@@ -0,0 +1,10 @@
+{
+  "eventsource-onmessage-trusted.any.js": {
+    "note": "An Event created in userspace can not be set to trusted.",
+    "skip": true
+  },
+  "format-field-event-empty.any.js": {
+    "note": "To be investigated.",
```

Comment:
If you investigated, do you know why this test is failing?

Reply:
Forgot to remove that case from the exception list.

Reply:
It is now also tested